google_takeout_parser/parse_html/html_time_utils.py

```diff
@@ -41,28 +41,64 @@ def abbr_to_timezone(abbr: str) -> tzinfo:
 _TIME_FORMAT = "%b %d, %Y, %I:%M:%S %p"
 
 
-def parse_html_dt(s: str) -> datetime:
```

Comment:
technically, this is an API change, so could default it to `None`, but don't think anything else is using it?

Reply:
yeah, I think its fine. Fine for there to be breaking changes for this project at this point, nothing really set in stone yet